docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -41,11 +41,11 @@ The main features of the `Tizen.Multimedia.Camera` class include:
     -   Flash
     -   Focus
     -   Metering
-    -   EXIF tag (geo, orientation, software info and description)
+    -   EXIF tag (geo, orientation, software information and description)
     -   Scene mode, HDR, theater
```

Comment:
Please correct the spelling of "theater" to "theatre"

Reply:
It's not a typo.
https://www.merriam-webster.com/dictionary/theater

Reply:
@hsgwon: 

I also explored a bit about it. Actually its the US and Europe English style difference. "Theatre" is European style and US style is "Theater". You may check this here: https://en.oxforddictionaries.com/definition/theatre

You may leave this point.

Reply:
Right, that's why I said it's not a typo. so I want to keep current style.
Please merge this PR if there's no other issues.